tests/zfs-tests/tests/functional/slog/slog_016_pos.ksh

```diff
@@ -0,0 +1,138 @@
+#!/bin/ksh -p
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2021 by Nutanix. All rights reserved.
+#
+
+. $STF_SUITE/tests/functional/slog/slog.kshlib
+
+#
+# DESCRIPTION:
+#	Verify saxattr logging in to ZIL works
+#
+# STRATEGY:
+#	1. Create an empty file system (TESTFS)
+#	2. Freeze TESTFS
+#	3. Create Xattrs.
+#	4. Unmount filesystem
+#	   <at this stage TESTFS is empty again and unfrozen, and the
+#	   intent log contains a complete set of deltas to replay it>
+#	5. Remount TESTFS <which replays the intent log>
+#	6. Check xattrs.
+#
+
+verify_runnable "global"
+
+log_assert "Verify saxattr logging in to ZIL works"
+log_onexit cleanup
+log_must setup
+
+NFILES=10
+function validate_zil_saxattr
+{
+	saxattrzil=$1
+
+	echo $saxattrzil > /sys/module/zfs/parameters/zfs_zil_saxattr_enabled
```

Comment:
Add the tunable to tests/zfs-tests/include/tunables.cfg and use set_tunable32 here.

Reply:
Done